src/genie/libs/parser/asa/show_inventory.py

```diff
@@ -49,14 +49,13 @@ def cli(self, output=None):
 
         # Name: "Chassis", DESCR: "ASA 5555-X with SW, 8 GE Data, 1 GE Mgmt"
         # Name: "power supply 1", DESCR: "ASA 5545-X/5555-X AC Power Supply"
-        p1 = re.compile(
-            r'^Name: +"+(?P<name>.+)"+,* +DESCR:+ "+(?P<description>.+)+"$')
+        p1 = re.compile(r'^Name:\s+"+(?P<name>.+)",\s+DESCR:\s"+(?P<description>.+)"$')
```

Comment:
```suggestion
        p1 = re.compile(r'^Name:\s+"(?P<name>.+)",\s+DESCR:\s+"(?P<description>.+)"$')
```

Reply:
looks like my typo, thanks for pointing that out